data/core/keymap.lua

```diff
@@ -35,6 +35,23 @@ local modkey_map = modkeys_os.map
 local modkeys = modkeys_os.keys
 
 
+---Normalizes a stroke sequence to follow the modkeys table
+---@param stroke string
+---@return string
+local function normalize_stroke(stroke)
+  local stroke_table = {}
+  for modkey in stroke:gmatch("(%w+)%+") do
+    table.insert(stroke_table, modkey)
+  end
+  if not next(stroke_table) then
+    return stroke
+  end
+  table.sort(stroke_table)
+  local new_stroke = table.concat(stroke_table, "+") .. "+"
+  return new_stroke .. stroke:sub(new_stroke:len() + 1)
+end
```

Comment:
What do you think about something like `gmatch("([^+]+)+?")`?
So that you don't need to do the `next` thing and the `sub` thing.